test/integration/two_step_verification_test.rb

```diff
@@ -92,6 +92,17 @@ class TwoStepVerificationTest < ActionDispatch::IntegrationTest
           assert SUCCESS, last_email.subject
         end
       end
+
+      should "accept a valid code from a device which has a small time lag" do
+        old_code = Timecop.freeze(35.seconds.ago) { ROTP::TOTP.new(@new_secret).now }
```

Comment:
There will be points where this will fail with 35 seconds as well. The drift only allows for 30 seconds, so if this test runs in the first 5 seconds of the 30 second 2SV window, it'll fail. This needs to be 30 seconds max, or we should change the `MAX_2SV_DRIFT_SECONDS` to something else.